unicore/distribute/utils.py

```diff
@@ -267,28 +279,26 @@ def format_content_type_object(repo, content_type, uuid):
         raise NotFound('Object does not exist.')
 
 
-def save_content_type_object(repo, schema, uuid, data):
+def save_content_type_object(workspace, schema, uuid, data):
     """
     Save an object as a certain content type
     """
-    storage_manager = StorageManager(repo)
     model_class = deserialize(schema,
                               module_name=schema['namespace'])
     model = model_class(data)
-    commit = storage_manager.store(model, 'Updated via PUT request.')
-    return commit, model
+    workspace.save(model, 'Updated via PUT request.')
+    return model
 
 
-def delete_content_type_object(repo, content_type, uuid):
+def delete_content_type_object(workspace, content_type, uuid):
     """
     Delete an object of a certain content type
     """
-    storage_manager = StorageManager(repo)
-    schema = get_schema(repo, content_type).to_json()
+    schema = get_schema(workspace.repo, content_type).to_json()
```

Comment:
I'm not sure how to handle schema/model class stuff. Some of the endpoints assume that the schemas as stored in the repo, which is not the case for our content repos. Do we update our content repos? Or do we fall back on models defined in `unicore.content`? Maybe I'm misunderstanding something.